e2e-tests/cypress/integration/appeal-submission-tasklist.feature

```diff
@@ -1,33 +1,52 @@
+      | "About you - Your details"                      |
+      | "Planning application - Application number"     |
+      | "Planning application - Upload application"     |
+      | "Planning application - Upload decision letter" |
+      | "Your appeal - Appeal statement"                |
+      | "Your appeal - Supporting documents"            |
+      | "Appeal site - Site location"                   |
+      | "Appeal site - Site ownership"                  |
+      | "Appeal site - Site access"                     |
+      | "Appeal site - Site safety"                     |
 
+  Scenario Outline: All the tasks are completed
+    Given an appeal has been made
+    When the <task> part of the appeal are completed
+    Then the status of the <task> is "COMPLETED"
     Examples:
-      | name        | email            | status        |
-      | ""          | ""               | "IN PROGRESS" |
-      | ""          | "good@email.com" | "IN PROGRESS" |
-      | "Good Name" | ""               | "IN PROGRESS" |
-      | "Good Name" | "good@email.com" | "COMPLETED"   |
+      | task                                            |
+      | "About you - Your details"                      |
+      | "Planning application - Application number"     |
+      | "Planning application - Upload application"     |
+      | "Planning application - Upload decision letter" |
+      | "Your appeal - Appeal statement"                |
+      | "Your appeal - Supporting documents"            |
+      | "Appeal site - Site location"                   |
+      | "Appeal site - Site ownership"                  |
+      | "Appeal site - Site access"                     |
+      | "Appeal site - Site safety"                     |
 
 
-  Scenario Outline: The status of the Your Details section is accurate
-    Given the user "is not" the original appellant
-    And the user has provided their name: <name> and email: <email>
-    And the user has provided <on behalf of> as the name of the original appellant
-    When the user checks the status of their appeal
-    Then the user should see that the "Your Details" task is <status>
+  Scenario: The status of the "Your details" task is IN PROGRESS
+    Given confirmation of whether appellant is original applicant is requested
+    When confirmation provided that appellant is original applicant
+    Then the status of the "About you - Your details" is "IN PROGRESS"
 
-    Examples:
-      | name                | email            | on behalf of | status        |
-      | "Another Good Name" | "good@email.com" | ""           | "IN PROGRESS" |
-      | "Another Good Name" | "good@email.com" | "Good Name"  | "COMPLETED"   |
+  Scenario: The status of the "Site ownership" task is IN PROGRESS
+    Given the site ownership is presented for the first time
+    When it is confirmed that the whole site is not owned
+    Then the status of the "Appeal site - Site ownership" is "IN PROGRESS"
```

Comment:
I think this may need additional scenarios to cover at least the two examples from the previous examples table but maybe even more scenarios if we want to cover all possibilities.

Reply:
> I think this may need additional scenarios to cover at least the two examples from the previous examples table but maybe even more scenarios if we want to cover all possibilities.

It's not my understanding to cover all the possibilities as for example the original appellant name is not supposed to be answered if the appellant did not answered not being the original appellant before. If we try to do this, it will trigger an appeal-service-api inconsistency error so it's really a bigger problem/topic. 

But we could raise this point at the daily or wait for an other view on the question.

Reply:
I agree that it is best for this feature file to focus only on displaying correct status badge so we could remove the scenarios that consider business rules to get into a particular state.